edx-platform/wikilearn/lms/templates/footer.html

```diff
@@ -23,16 +23,7 @@
       <nav class="nav-colophon" aria-label="${_('About')}">
         <ol>
           <li class="nav-colophon">
-            <a href="#">Privacy Policy</a>
-          </li>
-          <li class="nav-colophon">
-            <a href="#">About Wikimedia</a>
-          </li>
-          <li class="nav-colophon">
-            <a href="#">Disclaimers</a>
-          </li>
-          <li class="nav-colophon">
-            <a href="#">Cookie Statement</a>
```

Comment:
comment these lines as we are going to use them in the future.

Reply:
ok